version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.1"
```

Comment:
Ok I should have read the sbt-release README before pushing/saying anything, sorry !